docs/development/setup-vagrant.md

```diff
@@ -94,112 +94,38 @@ Now you are ready for [Step 2: Get Zulip Code.](#step-2-get-zulip-code).
 
 #### Ubuntu
 
-If you're in a hurry, you can copy and paste the following into your terminal
-after which you can jump to [Step 2: Get Zulip Code](#step-2-get-zulip-code):
-
-```
-sudo apt-get -y purge vagrant && \
-wget https://releases.hashicorp.com/vagrant/2.2.4/vagrant_2.2.4_x86_64.deb && \
-sudo dpkg -i vagrant*.deb && \
-sudo apt-get -y install build-essential git ruby lxc lxc-templates cgroup-lite redir && \
-vagrant plugin install vagrant-lxc && \
-vagrant lxc sudoers
-```
-
-For a step-by-step explanation, read on.
-
-##### 1. Install Vagrant
-
-For Ubuntu 18.04 Bionic or newer, you can just install from `apt`:
-
-```
-sudo apt install vagrant
-```
-
-For Ubuntu 16.04 Xenial and 14.04 Trusty, you'll need a more recent
-version of Vagrant than what's available in the official Ubuntu
-repositories.
-
-First uninstall any vagrant package you may have installed from the Ubuntu
-repository:
-
-```
-christie@ubuntu-desktop:~
-$ sudo apt-get purge vagrant
-```
-
-Now download and install the .deb package for [Vagrant][vagrant-dl].  E.g.:
+##### 1. Install Vagrant, Docker, and Git
```

Comment:
Do the problems with older Vagrant in Xenial that cause us to tell people to manually install not apply here?  I'm pretty sure they do.

Reply:
As far as I can tell from the Git history, the problems that originally motivated the Vagrant upgrade were with the older Vagrant in Trusty and a specific version only in Artful. I am working on setting up Xenial and Bionic VMs to test this on and will update as necessary when I do.